SeleniumPractice/src/test/java/SeleniumDevTest.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.ArrayList;
+
+public class SeleniumDevTest {
+
+    @Test
+    public void lastVersion_3_141_59() throws InterruptedException {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+
+        driver.manage().window().maximize();
+        driver.get("https://www.selenium.dev/");
+        Thread.sleep(1000);
+
+        WebElement download = driver.findElement(By.xpath("//*[@id=\"navbar\"]/a[1]"));
```

Comment:
This locator will fail as soon as some ordering or basic structural changes happen in the navigation bar.
Also, you can simply use single quotes and avoid unreadable back slashes

Reply:
Any other path brings to result when **documentation** is being opened. When I write it in DevTool it refers to download, but only not when it work in idea. I don't know how to fix this. I tried with 3 or 4 other paths

Reply:
it comes too right side.